src/test/java/picard/fingerprint/CrosscheckFingerprintsTest.java

```diff
@@ -791,6 +792,99 @@ public void testCheckFiles(final List<File> files, final int expectedRetVal, fin
         doTest(args.toArray(new String[0]), metrics, expectedRetVal, numberOfSamples , CrosscheckMetric.DataType.FILE, ExpectAllMatch);
     }
 
+    // Test mostly same as last test, but with explicit index paths given
```

Comment:
This test looks thorough for when the index files are in separate locations and provided, but you might also add a case where the index file map has missing elements to make sure the correct exception is thrown.

Reply:
Working on this still

Reply:
Just added a test that looks for the right exception in the case that `INPUT_INDEX_FORCE` is turned on but index paths are not provided (for files without indices next to them).

Reply:
I like this solution. I think we should turn on INPUT_INDEX_FORCE in production since to safeguard against running without index files in the future.

Reply:
Definitely agree!